### ISSUE TITLE
No HTML is generated

### ISSUE THREAD

**Issue by jensgro** [trys/eleventy-design-system#1]
I tried this interesting app but unfortunately no components are rendered. There is neither a button nor a quote.
I am using node v16 and everything was installed fast and without any errors. There are even no errors in the browser's console. 
As I am using your meta-macro in a private project (yet to be pubslished) I know that it works. So I am puzzled.

Regards, Jens

**Comment by jensgro** [comment]
The generated code lacks the information of the component:

````

{{ c("", {
  "label": "Hello, world"
}) }}

````

**Comment by khawkins98** [comment]
@jensgro are you on windows? for me I traced it back to a path issue in components.js

```js
  const path = fileObj.path.split('/');
  tree.components.push({
    ...fileObj.exports,
    name: path[path.length - 2]
  });
```

In #2 there is a fix that gets this working as expected on windows.

**Comment by jensgro** [comment]
@khawkins98  Yes, you are right. I am on Windows 10 and I tried your modifications. Everything works as planned. Great!!!
Now I can experiement with your solution.

**Comment by jensgro** [comment]
This problem is solved in Ticket #2.
